Function.cs

```diff
@@ -59,6 +59,9 @@ public class Function
             "emoji yellow",
             "simpsons yellow",
             "Amazon orange",
+            "peacock blue",
+            "creamson red",

```

Comment:
I'm thinking this should be crimson, unless is creamson is just a color I'm not familiar with?
```suggestion
            "crimson red",
```